### PR TITLE
Added Docker deployment support

### PR DESCRIPTION
**Pull request by sbehrends**
Added Dockerfile for deployment.
Thought is not necessary for development, since you just used yeoman.

### CODE REVIEW

File: app/templates/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+bower_components
```
Comment (neilhem) [inline app/templates/.dockerignore:2]: Should be an empty newline at the end of file

File: app/templates/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# Bundle app source
+COPY . /usr/src/app
+
+# Build app
+RUN gulp build
+
+# Serve app
+EXPOSE 9000
+CMD [ "gulp", "serve:dist" ]
```
Comment (neilhem) [inline app/templates/Dockerfile:24]: Should be an empty newline at the end of file

**Comment by neilhem** [comment]
Hi, thank you for your request. Would you mind to add commit with fixes, so we will be able to merge your PR?

**Comment by sindresorhus** [comment]
I think this makes more sense as a recipe than an option. We're not opinionated on how you distribute the built app.

**Comment by silvenon** [comment]
I agree with @sindresorhus, it would be great as a recipe!

**Comment by sbehrends** [comment]
Thanks for the feedback. I will create a recipe then and submit a new PR.

**Pull request closed without merging**